<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AchievementsWindow</class>
 <widget class="QWidget" name="AchievementsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>658</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tram17</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton 
{ 
	font: 75 14pt &quot;DS Eraser Cyr&quot;;
background-color: rgb(212, 212, 212);
border-style: outset;
border-radius: 15px;
border-width: 3px;
border-color: white;
padding: 7px;
}
QPushButton:hover
{ 
background-color: black;
color: white;
 }
QPushButton:pressed
{
border-style: inset;
}
QWidget#AchievementsWindow {
background-color: rgb(212, 212, 212);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>DS Eraser Cyr</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Достижения</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QFrame" name="verticalFrame">
     <property name="styleSheet">
      <string notr="true">QLabel:hover
{
color:red
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="Burn_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="Explode_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="Hit_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="Sad_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="Trip_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="Happy_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="TrueEnd_Ach">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>??????</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
